src/eddington_gui/app.py

```diff
@@ -244,6 +244,11 @@ def save_to_output_dir(self, widget):
         if self.fit_result is None:
             self.show_nothing_to_plot()
             return
+        if self.output_directory_input.value == "":
+            self.main_window.error_dialog(
+                title="Save error", message="No output dir was chosen",
```

Comment:
I would change the title to a more meaningful name such as"Results output save error", or something like that,

Reply:
Ok, changed in last commit 👍